setup.py

```diff
@@ -1,18 +0,0 @@
-import setuptools
-
-with open('requirements.txt') as f:
-    requirements = f.read().splitlines()
-
-setuptools.setup(
-    name='python-elemental',
-    author='CBS Interactive',
-    author_email='video-processing-team@cbsinteractive.com',
-    version='0.4',
-    include_patckage_data=True,
-    url='https://github.com/cbsinteractive/elemental.git',
-    license='MIT',
-    long_description=open('README.md').read(),
-    install_requires=requirements,
```

Comment:
Those deps were impacting Propeller... I split the dev and required dependencies now in Poetry.